mass_mailing_switzerland/models/mass_mailing_campaign.py

```diff
@@ -55,6 +57,18 @@ def open_unsub(self):
     def open_clicks(self):
         return self.mass_mailing_ids.open_clicks()
 
+    @api.model
+    def fields_view_get(self, view_id=None, view_type='form', toolbar=False,
```

Comment:
For what purpose did you add this function? I'm not sure it is needed.

Reply:
Old fix tentative that I forgot to remove, sorry !